SonarQube.Client/SonarQubeService.cs

```diff
@@ -329,6 +329,19 @@ public async Task<SonarQubeQualityProfile> GetQualityProfileAsync(string project
                     request.ProjectKey = projectKey;
                 }, token);
 
+        public Uri GetViewIssueUrl(string projectKey, string issueKey)
+        {
+            EnsureIsConnected();
+
+            // The URL should be in the same form as the permalink generated by SQ/SC e.g
+            // SonarQube : http://localhost:9000/project/issues?id=security1&issues=AXZRhxr-9W_phHQ8Bzgn&open=AXZRhxr-9W_phHQ8Bzgn
+            // SonarCloud: https://sonarcloud.io/project/issues?id=sonarlint-visualstudio&issues=AW-EuNQbXwT7-YPcXpll&open=AW-EuNQbXwT7-YPcXpll
+            // Versioning: so far the format of the URL is the same across all versions from at least v6.7
```

Comment:
I tested on v6.7; I didn't test earlier versions because they have been out of support for years so I didn't think it was worth the effort (I didn't want to spend the time rolling back to Java 2 to run earlier versions).